static/app/components/events/breadcrumbs/breadcrumbsDrawer.tsx

```diff
@@ -265,20 +265,20 @@ const VisibleFocusButton = styled(Button)`
     0 0 1px;
 `;
 
-const BreadcrumbDrawerContainer = styled('div')`
+export const EventDrawerContainer = styled('div')`
   height: 100%;
   display: grid;
   grid-template-rows: auto auto 1fr;
 `;
```

Comment:
What we if we put all these newly minted `<Event...>` helpers into their own file called `eventDrawer.tsx` or something like that. Then BreadCrumbDrawer and FeatureFlagDrawer could both import what they need (seems like lots of stuff). The two specific drawer implementations would be clearly siblings, sharing from a third place.

Reply:
yup, can do